src/zb_links/api/link/links.py

```diff
@@ -31,17 +31,17 @@ class LinkCollection(Resource):
     @api.marshal_list_with(link)
     @api.doc(
         params={
-            "author": {
+            "authors": {
```

Comment:
Technically, this is breaking change to the API. According to semver, you need to increase the API version number.